inputs/disk/disk_axi.in

```diff
@@ -72,7 +72,6 @@ reconstruct = plm
 riemann = hlle
 dfloor = 1.0e-10
 pfloor = 1.0e-13
-de_switch = 1.0
```

Comment:
Is this why the CI is breaking? Default `de_switch` value is `0.0` so this is changing behavior I think.

Reply:
yes, it passes with a small value

Reply:
btw, the darwin CI script as is makes it hard to diagnose what goes wrong because the log file and all outputs are unreachable

Reply:
Yeah you're right, I [opened an issue](https://github.com/lanl/artemis/issues/16) for me to update the script to store artifacts along with the submission job output file.